spec/lib/engine/games/config_spec.rb

```diff
@@ -1,10 +1,13 @@
 # frozen_string_literal: true
 
 require './spec/spec_helper'
+require './lib/engine/config/tile'
 
 require 'json'
 
 module Engine
+  include Engine::Config::Tile
+  tile_colors = COLORS
```

Comment:
Is there a cleaner way of getting hold of the `Engine::Config::Tile.COLORS` array? This works but it also includes everything else from the Tile module.

Reply:
Down in the spec you could just reference `Engine::Config::Tile::COLORS`. You still need to `require` the whole file, but then you don't need to `include` the module.

Reply:
Thanks for the suggestion, that's much better.